drivers/vulkan/vulkan_context.cpp

```diff
@@ -255,7 +257,7 @@ Error VulkanContext::_get_preferred_validation_layers(uint32_t *count, const cha
 		return OK;
 	}
 
-	VkLayerProperties *instance_layers = (VkLayerProperties *)malloc(sizeof(VkLayerProperties) * instance_layer_count);
+	auto *instance_layers = static_cast<VkLayerProperties *>(malloc(sizeof(VkLayerProperties) * instance_layer_count));
```

Comment:
In general, we do not use `auto` in the Godot code (with exception to templates), and it's definitely no adding readability.

Reply:
Yes, but in this case, it's obvious, because after the assignment we cast to this type.

Reply:
We don't allow the `auto` keyword anywhere in the Godot codebase, so please remove it: https://docs.godotengine.org/en/stable/community/contributing/cpp_usage_guidelines.html#auto-keyword

Reply:
> [We don't allow the `auto` keyword anywhere in the Godot codebase](https://docs.godotengine.org/en/stable/community/contributing/cpp_usage_guidelines.html#auto-keyword), so please remove it.

Alright.